package.json

```diff
@@ -71,6 +72,7 @@
     "husky": "^8.0.2",
     "less": "^4.1.2",
     "less-loader": "^7.3.0",
+    "mocha-multi-reporters": "^1.5.1",
```

Comment:
This is the wrong file to add your new packages to, everything should go into the `tests/lib/screenshot-testing/package.json`.

Reply:
Thank you @mneudert for not only pointing this out but also with the code to enable mocha-multi-reports correctly. Updated run-tests.js and config.dist.js based on your review.